chalice/app.py

```diff
@@ -765,3 +779,87 @@ class AuthRoute(object):
     def __init__(self, path, methods):
         self.path = path
         self.methods = methods
+
+
+class EventSource(object):
+    def __init__(self, name, handler_string):
+        self.name = name
+        self.handler_string = handler_string
+
+
+class CloudWatchEventSource(EventSource):
+    def __init__(self, name, handler_string, schedule_expression):
+        super(CloudWatchEventSource, self).__init__(name, handler_string)
+        self.schedule_expression = schedule_expression
+
+
+class ScheduleExpression(object):
+    def to_string(self):
+        raise NotImplementedError("to_string")
+
+
+class Rate(ScheduleExpression):
+    MINUTES = 'MINUTES'
+    HOURS = 'HOURS'
+    DAYS = 'DAYS'
+
+    def __init__(self, value, unit):
+        self.value = value
+        self.unit = unit
+
+    def to_string(self):
+        unit = self.unit.lower()
+        if self.value == 1:
+            # Remove the 's' from the end if it's singular.
+            # This is required by the cloudwatch events API.
+            unit = unit[:-1]
+        return 'rate(%s %s)' % (self.value, unit)
+
+
+class Cron(ScheduleExpression):
```

Comment:
Oh so I have to import those two from `chalice.app` right now, but I'm use to importing all of the user facing classes from `chalice` instead when I right my application.